docs/UserGuide.md

```diff
@@ -1,3 +1,23 @@
+# User Guide
+
+TABS helps project team leaders **overview tasks assigned to group members**. It is targeted at individuals with less programming background i.e. **less CLI-intensive** in nature.
+
+- Quick start
+- Features
+  - Adding a person: adduser
+  - Listing all persons: list
+  - Edit a person: edituser
+  - Locating persons by name: finduser
+  - Deleting a person : deleteuser
+  - Adding a group: addgroup
+  - Add member to  group: addmember
+  - Assign member a task: assigntask
+  - Locate group by name: display
+  - Delete a group: deletegroup
+  - Exiting the program : exit
+- Command summary
```

Comment:
Suggest adding:
1. Backticks around commands e.g. `adduser` in the first feature
2. Links to jump to relevant part. Will check up on how to do this.

Reply:
@LianGuoYang has code showing how to jump to relevant part in his [PR](https://github.com/AY2223S1-CS2103T-W10-1/tp/pull/21/files)

Relevant part is here:
~~~GFMD
5. Refer to the [Features](#features) below for details of each command.
~~~